README.md

```diff
@@ -1,14 +1,87 @@
-**Microverse_trial_3**
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# TESLA CLONE 
+
+> Description to the project
```

Comment:
[OPTIONAL] Please indicate the description of the project. This will help other developers to know what is all about your project.

Reply:
I noticed a little typo toward the end of this sentence. You might want to download Grammarly. It helps with any typos in your sentences and makes suggestions for corrections. This is important in order to help pass a clear and concise review to the students